promdump/promdump.go

```diff
@@ -24,8 +27,9 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/dsnet/compress/bzip2"
```

Comment:
Why this third party library instead of the [standard library's `compress/bzip2`](https://pkg.go.dev/compress/bzip2)?

Reply:
used third-party library because the standard library( bzip2 package) does not have a NewWriter function. The bzip2 package is used for decompression, not compression.